R/read_sc_all_home_care.R

```diff
@@ -31,15 +31,28 @@ read_sc_all_home_care <- function(sc_dvprod_connection = phs_db_connection(dsn =
     # fix 2017
     dplyr::mutate(
       financial_quarter =
-        dplyr::if_else(.data$financial_year == 2017L & is.na(.data$financial_quarter),
+        dplyr::if_else(
+          .data$financial_year == 2017L & is.na(.data$financial_quarter),
           4,
```

Comment:
```suggestion
          4L,
```
should financial quarter be an integer here?

Reply:
Just looking at the code I think we can avoid reading in `financial_quarter` completely, as we correct it here but it's never used for anything, so we may as well just not read it in the first place!